charts/rancher-gatekeeper-operator/v0.1.0/questions.yaml

```diff
@@ -1 +1,2 @@
-rancher_min_version: 2.4.0-rc1
\ No newline at end of file
+rancher_min_version: 2.4.0-rc1
+io.rancher.certified: experimental
```

Comment:
this needs to be nested under labels @prachidamle

Reply:
@deniseschannon rancher_min_version as well?